pytest_plugins/metadata_markers.py

```diff
@@ -103,6 +116,12 @@ def pytest_collection_modifyitems(session, items, config):
             doc_assignee = assignee_regex.findall(docstring)
             if doc_assignee and 'assignee' not in item_mark_names:
                 item.add_marker(pytest.mark.assignee(doc_assignee[0]))
+            doc_baseos = baseos_regex.findall(docstring)
+            if doc_baseos and 'BaseOS' not in item_mark_names:
+                item.add_marker(pytest.mark.BaseOS(doc_baseos[0]))
+            else:
+                rhel_version = get_sat_rhel_version().base_version
+                item.add_marker(pytest.mark.BaseOS(rhel_version))
```

Comment:
Remove this as Satellite BaseOS marker wont help with any purpose for now or in recent future.